test/e2e/main_test.go

```diff
@@ -194,7 +194,7 @@ var examplePaasConfig = v1alpha1.PaasConfig{
 			"default": {"admin"},
 			"viewer":  {"view"},
 		},
-		Whitelist: v1alpha1.NamespacedName{
+		GroupSyncList: v1alpha1.NamespacedName{
 			Namespace: "wlns",
```

Comment:
this implied WhiteList namespace. Might change to `gsns` (GroupSyncNameSpace) as well, combined with the content of the next comment.